yjit/src/core.rs

```diff
@@ -1420,9 +1420,6 @@ fn gen_block_series_body(
             .incoming
             .push(last_branchref.clone());
 
-        // This block should immediately follow the last branch
-        assert!(new_blockref.borrow().start_addr == last_branch.end_addr);
```

Comment:
memo: This no longer holds because the end of the block is now not `last_branch.end_addr` but `nop`s after the branch.